src/app/profile/page.tsx

```diff
@@ -0,0 +1,14 @@
+"use client";
+
+import { useCurrentUser } from "@/domains/user/hooks";
+import Loading from "./user-info/loading";
+import UserInfo from "./user-info";
+
+const UserPage = () => {
+  const { data } = useCurrentUser();
+  const user = data?.data;
+
+  return <>{user ? <UserInfo user={user} /> : <Loading />}</>;
```

Comment:
There's no need of `Fragment` wrapper.
```suggestion
  return user ? <UserInfo user={user} /> : <Loading />;
```